docs/source/project_documentation/user_guide/model.md

```diff
@@ -0,0 +1,83 @@
+# Model Definitions in AaC
+
+The AaC `model` root key is used to represent the components of a system. These,
+for example, can be the services in a system following a Microservice
+Architecture (MSA) design.
+
+In its most basic form, a `model` must have a `name` that distinguishes it from
+other AaC definitions. At the same time, however, a `model` that defines nothing
+more than a `name` is of little use for anything more than representing external
+components or actors.
+
+It is recommended to include some amount of `behavior` definitions for each
+model as that allows users and the AaC package to understand the component as a
+part of the system. Some questions we can answer for `model`s with defined
+`behavior`s are:
```

Comment:
These behaviors can also be applied to the overall system model as opposed to just its components.